test/uk/gov/hmrc/uknwauthcheckerapi/services/ValidationServiceSpec.scala

```diff
@@ -88,41 +91,112 @@ class ValidationServiceSpec extends BaseSpec {
       }
     }
 
-    "return JsError when AuthorisationRequest has too many Eoris" in {
-      forAll { (tooManyEorisRequest: TooManyEorisAuthorisationRequest) =>
-        val json = Json.toJson(tooManyEorisRequest.request)
+    "must return JSError with usual case content when maxEori is 3000" when {
+      "AuthorisationRequest has too many Eoris" in {
+        forAll { (tooManyEorisRequest: TooManyEorisAuthorisationRequest) =>
+          val json = Json.toJson(tooManyEorisRequest.request)
 
-        val expectedResponse =
-          ValidationDataRetrievalError(
-            JsError(
-              Seq((JsPath \ JsonPaths.eoris, Seq(JsonValidationError(ApiErrorMessages.invalidEoriCount))))
+          val expectedResponse =
+            ValidationDataRetrievalError(
+              JsError(
+                Seq((JsPath \ JsonPaths.eoris, Seq(JsonValidationError("The request payload must contain between 1 and 3000 EORI entries"))))
```

Comment:
Can you utilise the ApiErrorMessages. We don't want literals where we can avoid it.